ranger2/aws/ranger_asg/terraform.tfvars

```diff
@@ -0,0 +1,97 @@
+access_key = "XXXX"  
+secret_key = "XXXX"
+
+aws_session_token = "XXXX"
+
+region = "us-east-1"
+
+vpc_id = "vpc-XXXX"
+public_subnets = ["subnet-XXXX", "subnet-XXXX"]
+private_subnets = ["subnet-XXXX", "subnet-XXXX"]
+#your local ip
+local_ips = ["99.0.0.0/32"]
```

Comment:
whose local IP is this ? the person who is setting this up ?

Reply:
This IPs will be connecting AWS resource from like LBs or ssh into instance.